quickwit/quickwit-indexing/src/source/mod.rs

```diff
@@ -117,7 +118,7 @@ impl SourceExecutionContext {
     ) -> Arc<SourceExecutionContext> {
         Arc::new(Self {
             metastore,
-            index_id: index_id.to_string(),
+            index_uid: IndexUid::new(index_id),
```

Comment:
That's not what I meant, but considering the current usage, that will work for now.